### DISCUSSION TITLE
Update without scrolling

### DISCUSSION THREAD

**Discussion by phikal** [cheeaun/phanpy#738] [Ideas]
I hope I am not just missing an option, but I'd like to enable auto-update without having the view scroll.  So whenever I return back to the phanpy tab, I can just scroll up to see what's new.   I know of the "Catch Up" option, but what I had in mind would fit more into the main view.

**Comment by jcarnat** [comment]
Hi,

Just adding a +1 here.
I like using Phanpy but loosing track of the current position in read/unread TL is quite annoying.

I believe both https://github.com/cheeaun/phanpy/issues/269 and https://github.com/cheeaun/phanpy/issues/741 points towards this direction.

Would be great if something like this would be implemented. It's the only "missing feature" that prevents me from using it on iOS rather than using an App :)
